Mathlib/Analysis/Calculus/ParametricIntegral.lean

```diff
@@ -165,6 +166,28 @@ theorem hasFDerivAt_integral_of_dominated_loc_of_lip {F' : α → H →L[𝕜] E
   apply hasFDerivAt_integral_of_dominated_loc_of_lip' δ_pos <;> assumption
 #align has_fderiv_at_integral_of_dominated_loc_of_lip hasFDerivAt_integral_of_dominated_loc_of_lip
 
+/-- Interval version of `hasFDerivAt_integral_of_dominated_loc_of_lip` -/
+theorem hasFDerivAt_integral_of_dominated_loc_of_lip_interval [NormedSpace ℝ H] {μ : Measure ℝ}
+    {F : H → ℝ → E} {F' : ℝ → H →L[ℝ] E} {a b : ℝ} {bound : ℝ → ℝ} (ε_pos : 0 < ε)
+    (hF_meas : ∀ᶠ x in 𝓝 x₀, AEStronglyMeasurable (F x) <| μ.restrict (Ι a b))
+    (hF_int : IntervalIntegrable (F x₀) μ a b)
+    (hF'_meas : AEStronglyMeasurable F' <| μ.restrict (Ι a b))
+    (h_lip : ∀ᵐ t ∂μ.restrict (Ι a b),
+      LipschitzOnWith (Real.nnabs <| bound t) (fun x ↦ F x t) (ball x₀ ε))
+    (bound_integrable : IntervalIntegrable bound μ a b)
+    (h_diff : ∀ᵐ t ∂μ.restrict (Ι a b), HasFDerivAt (fun x ↦ F x t) (F' t) x₀) :
+    IntervalIntegrable F' μ a b ∧
+      HasFDerivAt (fun x ↦ ∫ t in a..b, F x t ∂μ) (∫ t in a..b, F' t ∂μ) x₀ := by
```

Comment:
I would split these into two lemmas.

Reply:
Sure, I can do this - but since this matching the existing lemmas in this file, I would perform the same split to `hasFDerivAt_integral_of_dominated_loc_of_lip` and `hasFDerivAt_integral_of_dominated_loc_of_lip'`.

Reply:
I just pushed a commit doing so. (Sorry for the force push. Only the last five commits are new.)
I'm out of Lean time for today; I can write docstrings for the new lemmas when I return to this (not before next weekend).